Sources_v3/StreamChat/ChatClient_Tests.swift

```diff
@@ -10,14 +10,21 @@ class ChatClient_Tests: StressTestCase {
     var userId: UserId!
     private var testEnv: TestEnvironment<DefaultExtraData>!
     
-    // A helper providing ChatClientConfic with in-memory DB option
+    // A helper providing ChatClientConfig with in-memory DB option
     var inMemoryStorageConfig: ChatClientConfig {
         var config = ChatClientConfig()
         config.isLocalStorageEnabled = false
         config.baseURL = BaseURL(urlString: .unique)!
         return config
     }
     
+    // Helper for providing config with in-memory DB option and passive (inactive) mode
+    lazy var inactiveInMemoryStorageConfig: ChatClientConfig = {
+        var config = inMemoryStorageConfig
+        config.isClientInActiveMode = false
+        return config
+    }()
```

Comment:
nit: I don't think "passive" means the same thing as "inactive"

Reply:
Only in our context :D I opted for the boolean flag for simplicity, so anything inactive = passive (May Zeus protect me from grammar people)